### DISCUSSION TITLE
Return of Snow

### DISCUSSION THREAD

**Discussion by baitmooth** [baitmooth/snow#29] [Announcements]
After 2 Years Snow is back. There were some tasks to complete to reach a release.
We updated to the latest Candybar.
I did some new icons, mostly to get into flow again…
Great  @Donnnno  and awesome [@Kaserdragon2](https://github.com/Kaiserdragon2) helped a lot by sharing their build tools.

**Full Changelog**: https://github.com/baitmooth/snow/compare/1.0.0...1.3.8

<hr /><em>This discussion was created from the release <a href='https://github.com/baitmooth/snow/releases/tag/1.3.8'>Return of Snow</a>.</em>

**Comment by Donnnno** [comment]
Yessss, nice work!!
